src/reports/io/hdf5_writer.cpp

```diff
@@ -62,7 +62,7 @@ void HDF5Writer::write(double* buffer, int steps_to_write, int total_steps, int
         H5Sselect_hyperslab(space, H5S_SELECT_OR, &m_offset[i], NULL, &count[i], NULL);
     }*/
 
-    H5Dwrite(m_dataset, H5T_NATIVE_DOUBLE, memspace, filespace, H5P_DEFAULT, buffer);
+    H5Dwrite(m_dataset, H5T_NATIVE_DOUBLE, memspace, filespace, H5P_DEFAULT, &buffer[0]);
```

Comment:
I think you are looking for `buffer.data()`

Reply:
Yep :+1: